src/main/java/edu/project2/tripplanner/controller/CommentsController.java

```diff
@@ -0,0 +1,71 @@
+package edu.project2.tripplanner.controller;
+
+import edu.project2.tripplanner.model.Comment;
+import edu.project2.tripplanner.service.ICommentService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@AllArgsConstructor
+public class CommentsController {
+
+    private ICommentService iCommentService;
+
+    @ResponseBody
+    @GetMapping("/comments")
+    public List<Comment> findAllComments() {
+
+        return iCommentService.findAll();
+    }
+
+    @ResponseBody
+    @GetMapping("/user/{userId}/comments")
+    public List<Comment> getAllCommentsByUserId(@PathVariable(value = "userId") Long userId) {
+
+        return iCommentService.findByUserId(userId);
+    }
+
+    @ResponseBody
+    @GetMapping("/user/{userId}/comments/{commentId}")
+    public Optional<Comment> findCommentByIdAndUserId(@PathVariable(value = "userId") Long userId,
+                                                      @PathVariable(value = "commentId") Long commentId) {
+
+        return iCommentService.findByIdAndUserId(commentId, userId);
+    }
+
+
+    @ResponseBody
+    @PostMapping("/user/{userId}/place/{placeId}/comments")
```

Comment:
too long API

/ comment POST will be clear enough 

create DTO with fileds: comment, userId, placeId as Request body